src/lib.rs

```diff
@@ -1,22 +0,0 @@
-// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-//
-// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE-BSD-3-Clause file.
-//
-// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
-#![deny(missing_docs)]
```

Comment:
Should this lint be added to all the new crates?

Reply:
Yes it should. Good catch!

Reply:
Done